app/context/preferencesContext.js

```diff
@@ -0,0 +1,8 @@
+import {createContext} from 'react';
+
+export const PreferencesContext = createContext({
+  rtl: 'left',
```

Comment:
It is awesome to have a theme setup as context. it has a great future but I'm not ok with your name for direction, what does it mean `rtl: 'left'`? I read whole the code but think deeply, the code should be very readable and understandable for a normal developer in an unknown future.

If you are agreed on me, let change this naming.

Reply:
we have a language variable in redux ( store/app ) that it has a "isRTL" key and it used for handle change app language

i think is better to use that variable for now otherwise we have to  refactore and move related logic from redux to contex 

what is your opinions @ramyareye @amerllica

Reply:
Thank you Amer for your suggestion
I`m agree with @maleking  to use Redux to handle direction

Reply:
@maleking, Using context is great, but we already use **redux** for some global configuration. I'm agreed with you too to use redux instead of context API and now I have a question, Dear @Tohid1999, How you wanna change this config, Are you going to change your PR?